### ISSUE TITLE
Dev: Splashing runtime with bottles causes runtimes

### ISSUE THREAD

**Issue by Lohikar** [Aurorastation/Aurora.3#1359]
```
runtime error: 
[00:46:36]Division by zero
[00:46:36]proc name: trans to holder (/datum/reagents/proc/trans_to_holder)
[00:46:36]  source file: Chemistry-Holder.dm,260
[00:46:36]  usr: Bluespace Technician (/mob/living/carbon/human/bst)
[00:46:36]  src: /datum/reagents (/datum/reagents)
[00:46:36]  usr.loc: the floor (143,108,1) (/turf/simulated/floor/tiled/white)
[00:46:36]  call stack:
[00:46:36]/datum/reagents (/datum/reagents): trans to holder(/datum/reagents (/datum/reagents), 58.8, 0, 0)
[00:46:36]/datum/reagents (/datum/reagents): trans to mob(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), 58.8, 1, 0, 0)
[00:46:36]/datum/reagents (/datum/reagents): splash mob(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), 58.8, 0)
[00:46:36]/datum/reagents (/datum/reagents): trans to(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), 58.8, 1, 0)
[00:46:36]/datum/reagents (/datum/reagents): splash(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), 58.8, 1, 0, 0, 60)
[00:46:36]the dylovene bottle (/obj/item/weapon/reagent_containers/glass/bottle/antitoxin): standard splash mob(Bluespace Technician (/mob/living/carbon/human/bst), Runtime (/mob/living/simple_animal/cat/fluff/Runtime))
[00:46:36]the dylovene bottle (/obj/item/weapon/reagent_containers/glass/bottle/antitoxin): afterattack(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), Bluespace Technician (/mob/living/carbon/human/bst), 1, "icon-x=22;icon-y=9;left=1;scre...")
[00:46:36]Bluespace Technician (/mob/living/carbon/human/bst): ClickOn(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), "icon-x=22;icon-y=9;left=1;scre...")
[00:46:36]Bluespace Technician (/mob/living/carbon/human/bst): ClickOn(Runtime (/mob/living/simple_animal/cat/fluff/Runtime), "icon-x=22;icon-y=9;left=1;scre...")
[00:46:36]Runtime (/mob/living/simple_animal/cat/fluff/Runtime): Click(the floor (143,109,1) (/turf/simulated/floor/tiled/white), "mapwindow.map", "icon-x=22;icon-y=9;left=1;scre...")
```

**Comment by NanakoAC** [comment]
ill fix this tomorrow, i know of a couple unsolved issues in that code

**Comment by NanakoAC** [comment]
fixed in #1366
